pymchelper/executor/runner.py

```diff
@@ -151,18 +164,35 @@ def get_data(self):
         return estimators_dict
 
     def clean(self):
-        """
-        Removes all working directories (if exists)
-        """
+        """Removes all working directories (if exists)"""
         self.workspace_manager.clean()
 
+    @staticmethod
+    def __update_fluka_input_file(destination: str, rng_seed: int):
+        """Updates the FLUKA input file with the new RNG seed."""
+        configuration = Input.Input(destination)
+        cards : List[Input.Card] = configuration.cardlist
+        randomize = [(index, card) for index, card in enumerate(cards) if str(card.tag).startswith('RANDOMIZ')]
+
+        rng_card = Input.Card("RANDOMIZ")
+        rng_card.setComment("updated random number generator settings")
+        rng_card.setWhat(1, 1.)
+        rng_card.setWhat(2, rng_seed)
+
+        if randomize:
+            configuration.replaceCard(randomize[0][0], rng_card)
+        else:
+            configuration.addCard(rng_card)
+
+        configuration.write(destination)
```

Comment:
Take a look at https://github.com/DataMedSci/pymchelper/pull/656#issuecomment-1746912323

It seems that processing full input file with reading by `Input.Input(destination)`  and saving with `configuration.write(destination)` is producing a malformed output file. 

I'd rather try manual replacement of the lines in the file with content of appropiate card, rather tahn writing full file with `configuration.write(destination)`

Reply:
![obraz](https://github.com/DataMedSci/pymchelper/assets/128628266/1169c265-3bec-4208-a0cf-3ab823446e54)

Hope tests will pass now :)